src/PBot/TaskForces/InvolvedIssue.cs

```diff
@@ -0,0 +1,12 @@
+namespace PBot.TaskForces
+{
+    using Octokit;
+
+    public class InvolvedIssue
```

Comment:
@mauroservienti since this is not deployed for other people to reference the is no difference between `internal` and `public`